taipy/core/config/job_config.py

```diff
@@ -33,7 +33,7 @@ class JobConfig:
     _DEFAULT_MODE = "standalone"
 
     _MODE_TO_MODULE: Dict[str, str] = {
-        "airflow": "taipy.airflow.scheduler",
+        "airflow": "taipy.airflow",
         "enterprise": "taipy.enterprise.core.scheduler.scheduler",
     }
```

Comment:
How about creating 2 different dicts: one for mapping to the Config class module and one for Scheduler class (to be more explicit), as enterprise is not a job mode and we don't have Config for enterprise.

Reply:
ah yes exactly!! that's what I was talking about a "weird spot" in my approach. Have already pushed :D